src/filesystem.ts

```diff
@@ -8,10 +8,20 @@ import * as Versions from "./fs/versions.js"
 
 import FileSystem, { Dependencies } from "./fs/filesystem.js"
 
+import * as Ucan from "./ucan/index.js"
```

Comment:
Could we re-order the imports a bit? The order we have generally been using is:

- Imports from external libraries
- Imports from within Webnative

Within each of those:

- Namespace imports (e.g. `import * as name from "module-name"`)
- Named imports (e.g. `import { export1, export2 } from "module-name"`)
- Default imports (e.g. `import defaultExport from "module-name"`)

The imports in each of those should be alphabetized. 

For example: https://github.com/fission-codes/webnative/blob/730d8e1ae489bc1ed2a148ba8f756712b08c2b66/src/index.ts#L26-L53

Reply:
cool, are you thinking something more like this? 

```ts
import { CID } from "multiformats/cid"

import * as Crypto from "./components/crypto/implementation.js"
import * as Depot from "./components/depot/implementation.js"
import * as DID from "./did/index.js"
import * as Reference from "./components/reference/implementation.js"
import * as Storage from "./components/storage/implementation.js"
import * as Protocol from "./fs/protocol/index.js"
import * as Versions from "./fs/versions.js"
import * as RootKey from "./common/root-key.js"
import * as Ucan from "./ucan/index.js"

import { AuthenticationStrategy } from "./index.js"
import { Branch } from "./path/index.js"
import { Configuration } from "./configuration.js"
import { Maybe, decodeCID, EMPTY_CID } from "./common/index.js"
import { type RecoverFileSystemParams } from "./fs/types/params.js"

import FileSystem, { Dependencies } from "./fs/filesystem.js"
```

Reply:
edited that, it wasn't in alphabetical order the first time i did it 😅

Reply:
Yep, that looks good! ✨